aspnet/whitepapers/side-by-side-with-10.md

```diff
@@ -75,17 +75,17 @@ The Aspnet\_regiis.exe provides two options for script mapping a Web application
 
 The path defines the Web application IIS metadata path, which is defined in the form of W3SVC/ROOT/{WebSiteNumber}/{Application\_Name}. For example, for a Web application called Portal located under the default Web site, the metabase path is W3SVC/1/ROOT/Portal.
 
-![](side-by-side-with-10/_static/image4.gif)
+![Screenshot that shows a Web application called Portal located under the default Web site, and the metabase path is W3SVC/1/ROOT/Portal.](side-by-side-with-10/_static/image4.gif)
```

Comment:
"W3SVC/1/ROOT/Portal" might be difficult for a screen-reader to read out. Try writing out this part of the text, for example: W 3 S V C slash 1 slash ROOT slash Portal.